packages/components/src/DataList/components/DataListLayout/DataListLayout.tsx

```diff
@@ -1,8 +1,24 @@
 import React from "react";
-import { DataListItemType, DataListObject } from "../../DataList.types";
+import {
+  Breakpoints,
+  DataListItemType,
+  DataListObject,
+} from "../../DataList.types";
 
 export interface DataListLayoutProps<T extends DataListObject> {
   children: (item: DataListItemType<T[]>) => JSX.Element;
+
+  /**
+   * The breakpoint at which the layout should be displayed. It will be rendered until a layout with a larger breakpoint is found.
+   * @default "xs"
+   */
+  readonly size?: Breakpoints;
+
+  /**
+   * Determines if a header is shown for the layout
+   * @default true
+   */
+  readonly showHeader?: boolean;
```

Comment:
Thinking more about this. Might be better if it's handled through the `DataList` rather than the `Layout`. Like so `headerVisibility={{ xs: false, large: true }}` (prop name TBD).

Thinking about that scenario when someone only uses 1 layout and passes in a pure CSS responsive layout. They wouldn't be able to remove that header.

Happy to talk about this IRL